cli_tools_e2e_test/import/scratch_ownership_test.go

```diff
@@ -0,0 +1,156 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package import_test
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"testing"
+
+	"cloud.google.com/go/storage"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/api/googleapi"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/gce_vm_image_import/cli"
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools_e2e_test/common/config"
+)
+
+const (
+	// For this test to work, it needs to be run within
+	// a project *other* than compute-image-test-resources.
```

Comment:
Why is compute-image-test-resources project a factor here? Gcloud copies files from source bucket to scratch bucket, which, in the case of e2e tests, will be in compute-image-test-pool-001.

Reply:
I used `compute-image-test-resources` as the owner of the bucket so that the test could be run in any other project (a personal project for test, one of the pool projects, `compute image tools test`, etc.

Reply:
Makes sense. FYI, we don't (or shouldn't) run anything in compute-image-test-resources, it's only for storing resources.